lib/data/icons.dart

```diff
@@ -167,4 +167,8 @@ class GalleryIcons {
     0xe927,
     fontFamily: 'GalleryIcons',
   );
+  static const IconData navigationRail = IconData(
+    60080,
+    fontFamily: 'MaterialIcons',
```

Comment:
Could you please include a screenshot of what this new Icon looks like, since its coming from a different font family, I'd like to see how it looks.

Reply:
@rami-a , Please have a look at [this screenshot](https://drive.google.com/file/d/1AGnZVqLTx5xEdkS6mZYIY5fUBn2atIyZ/view) .
Please let me know if its needs to be changed.

Reply:
Cool, that looks good to me!